platforms/jvm/plugins-jvm-test-suite/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -1007,7 +1007,7 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
     }
 
     @Issue("https://github.com/gradle/gradle/issues/25604")
-    def "test suite configurations can be copied"() {
+    def "test suite configurations can not be copied"() {
```

Comment:
💭 We might even consider just deleting this test, since we already have test coverage saying you can't copy non-resolvable configurations. This might be unnecessary

Reply:
This coverage test is specifically for configurations created to support test suites.  The role of those configurations is an implementation detail and could change.  I think this test is worth keeping in case it does.